src/api/app/models/bs_request_action.rb

```diff
@@ -245,11 +246,12 @@ def notify_params(ret = {})
   end
 
   def contains_change?
-    sourcediff(nodiff: 1).present?
-  rescue BsRequestAction::Errors::DiffError
-    # if the diff can'be created we can't say
-    # but let's assume the reason for the problem lies in the change
-    true
+    spkg = Package.find_by_project_and_name(source_project, source_package)
+    tpkg = Package.find_by_project_and_name(target_project, target_package)
+
```

Comment:
`spkg` and `tpkg` variable names can be improved :thought_balloon:

Reply:
Done. I've updated the names. I wasn't able to use the complete names because those variables are names of attributes as well.